apps/end-tenancy/acceptance/features/property-address.js

```diff
@@ -175,6 +175,18 @@ Scenario('I am taken to the tenant-details page on a valid submission', (
   tenantDetailsPage
 ) => {
   propertyAddressPage.selectAddressAndSubmit();
-  I.submitForm();
   I.seeInCurrentUrl(tenantDetailsPage.url);
 });
+
+Scenario('I am taken to the tenancy-start page if I am requesting an nldp', function *(
```

Comment:
it's a generator function - https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/function*

Basically it tells codecept the function contains some async actions - in this case updating the session and refreshing the page - the yield keyword will pause execution until the async action has resolved